apps/cli/src/generator.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing'
+import { Tree, readProjectConfiguration } from '@nx/devkit'
+
+import { codeGenGenerator } from './generator'
+import { CodeGenGeneratorSchema } from './schema'
+
+describe('code-gen generator', () => {
+    let tree: Tree
+    const options: CodeGenGeneratorSchema = { name: 'test' }
+
+    beforeEach(() => {
+        tree = createTreeWithEmptyWorkspace()
+    })
+
+    it('should run successfully', async () => {
```

Comment:
It would be good to do some snapshot tests here

Reply:
Done